### DISCUSSION TITLE
CLI Check If Migrations Applied?

### DISCUSSION THREAD

**Discussion by AdamEisfeld** [prisma/prisma#18660] [Q&A] [answered]
Is there a way to check if the migrations in my local migrations folder have been applied to the DB in production?

I see the 

```prisma migrate diff --from-url --to-migrations```

Command in the reference (https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-diff) but it seems to require a "--shadow-database-url" flag (although the example on that page does not provide one). It seems to me this should be possible without. Doesn't "prisma migrate dev" determine if changes need to be made in its internal logic?

I realize I can use 

```prisma migrate diff --from-url --to-schema-datamodel```

And pass the location of my local data model, but that does not capture migrations that aren't a part of the data model (specifically, adding RLS / Policies in an additional custom migration I make).

I am trying to write an npm script that will tell me if my local state is out of sync with the db. Any advice?

Thanks, 
- Adam

**Comment by nurul3101** [comment, marked as answer]
Hello @AdamEisfeld 👋 

I think you are looking for `migrate status` command: [Reference](https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-status).

> The prisma migrate status command looks up the migrations in /prisma/migrations/* folder and the entries in the _prisma_migrations table and compiles information about the state of the migrations in your database.

One another possible solution to check if migration is applied could be to compare your local migration history with the migration history stored in the `_prisma_migrations` table in your production database. You can write an npm script to fetch and compare the local and remote migration history.

**Reply by nurul3101** [reply]
Hello @AdamEisfeld 👋 

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
